Tests/scripts/hook_validations/integration.py

```diff
@@ -66,9 +66,46 @@ def is_valid_integration(self):
         """Check whether the Integration is valid or not, update the _is_valid field to determine that"""
         self.is_valid_subtype()
         self.is_default_arguments()
+        self.is_proxy_configured_correctly()
+        self.is_insecure_configured_correctly()
```

Comment:
Not all of our integrations are configured that way so it will be problematic.
It will cause too much trouble

Reply:
we should consider it to be a standard for new integration...
for every `fetch-incident` we need a limit of incident per fetch (this is a must) and limit how much back to go (however this might not be by time so you are correct).